### ISSUE TITLE
Test flag is turned on by default

### ISSUE THREAD

**Issue by smaret** [smaret/lime#31]
ad69ff9451647f28cd75ce4fc7000aef192c3ca6 introduces a TEST flag that can used to test that LIME produces the same output in two consecutive runs. This is done by fixing the seed of the random number generator. 

However, this flag is turned on by default:

```shell
./Makefile:CCFLAGS = -O3 -falign-loops=16 -fno-strict-aliasing -DTEST -I${INCLUDEDIR_FITSIO} -I${INCLUDEDIR_QHULL} 
```

Obviously, this should not be the case.

**Comment by allegroLeiden** [comment]
Fair enough.

**Comment by allegroLeiden** [comment]
Should be closed with my latest commit.

**Comment by smaret** [comment]
Fixed with 83b1692478f1c680a3a0ae4a8b0aa866650e74e4.
